http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/package.scala

```diff
@@ -44,6 +45,13 @@ package object http {
       .compile
       .to(List)
 
+  def writeTempFile[F[_]: Sync: ContextShift](prefix: String, data: Array[Byte], blocker: Blocker): F[Path] =
```

Comment:
what is this for?

Reply:
Yeah.. as far as I can tell the Go client only accepts the cert as a file path. So I'm writing a temp file in Leo immediately before invoking helm. 

Not ideal but I think it should work..

Reply:
ah, bummer..okay..